src/containers/AccountSuccessTimeout.jsx

```diff
@@ -0,0 +1,69 @@
+import styles from '../styles/accountConnection'
+
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+import AccountConnectionData from '../components/AccountConnectionData'
+
+class AccountSuccessTimeout extends Component {
+  constructor (props, context) {
+    super(props, context)
+    this.store = this.context.store
+  }
+
+  cancel () {
+    this.props.onCancel()
+  }
+
+  // TODO: use a better helper
+  getIcon (konnector) {
+    try {
+      return require(`assets/icons/konnectors/${konnector.slug}.svg`)
+    } catch (error) {
+      console.warn(error.message)
+      return require('assets/icons/konnectors/default.svg')
+    }
+  }
+
+  render () {
+    const { t, account, connector, accountConfig } = this.props
```

Comment:
I do not like `onUpdate`, because I would expect that to be called on the save button after a change in the password, not to display the form that would allow me to perform the update.